src/main/java/io/cdap/e2e/utils/SeleniumHelper.java

```diff
@@ -131,4 +132,22 @@ public static boolean verifyElementPresent(String locator) {
       return false;
     }
   }
+
+  public static boolean waitForParticularTime(int time) throws InterruptedException {
+    Thread.sleep(time);
```

Comment:
We should not use sleeps, it just slows tests down. Please wait for specific state instead

Reply:
all sleep removed